edgedb/datatypes/tuple.c

```diff
@@ -51,16 +45,14 @@ EdgeTuple_New(Py_ssize_t size)
         return NULL;
     }
 
-    EdgeTupleObject *obj = NULL;
+    PyTupleObject *obj = NULL;
 
-    EDGE_NEW_WITH_FREELIST(EDGE_TUPLE, EdgeTupleObject,
+    EDGE_NEW_WITH_FREELIST(EDGE_TUPLE, PyTupleObject,
```

Comment:
Basically we don't need this line. Please double check, but IIRC the builtin tuple type already uses freelists.

Reply:
Yes it does: https://github.com/python/cpython/blob/3.7/Objects/tupleobject.c#L87-L110